server/models/productModel.js

```diff
@@ -5,79 +5,32 @@ const BASE_URL = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/products';
 const AUTH = {'Authorization': `${config.TOKEN}`};
 
 module.exports = {
```

Comment:
nice to reformat the request handling code, now it looks much cleaner! I should do the same!